browser/src/libs/bitbucket/scrape.ts

```diff
@@ -87,14 +87,27 @@ export const getCommitIDFromLink = (selector = 'a.commitid'): string => {
     return commitID
 }
 
+const getCommitIDFromRevisionSelector = (): string => {
+    const revisionSelectorSpan = document.querySelector<HTMLElement>('span[data-revision-ref]')
+    if (!revisionSelectorSpan) {
+        throw new Error('Could not find span[data-revision-ref] element')
+    }
+    try {
+        const { latestCommit }: { latestCommit: string } = JSON.parse(revisionSelectorSpan.dataset.revisionRef!)
+        return latestCommit
+    } catch (err) {
+        throw new Error(`Could not determine commitID from revision selector: ${err}`)
```

Comment:
This would `err.toString()`, which would include the whole stack trace (which is almost the same). The only error this can be is SyntaxError.
```suggestion
        throw new Error('Could not parse JSON from revision selector')
```